dwave_networkx/drawing/chimera_layout.py

```diff
@@ -230,87 +230,88 @@ def draw_chimera(G, **kwargs):
 
 
 def draw_chimera_embedding(G, *args, **kwargs):
-    """Draws an embedding onto the chimera graph G, according to layout.
+    """Draw an embedding onto the chimera graph ``G``, according to the Chimera layout.
 
-    If interaction_edges is not None, then only display the couplers in that
-    list.  If embedded_graph is not None, the only display the couplers between
-    chains with intended couplings according to embedded_graph.
+    If the ``interaction_edges`` parameter is not None, then only display the couplers in that
```

Comment:
Do we need this intro paragraph? Why can't this all be specified in the descriptions of the parameters themselves? The intro should be to point out general, crucial, or unintuitive aspects of the function